### PR TITLE
add allItems / Browse section

### PR DESCRIPTION
**Pull request by ChrisEddy**

### CODE REVIEW

File: components/App.js [hunk 1]
```diff
@@ -158,7 +158,8 @@ export const toggleMenus = props => {
         activeTab: appState.menu.activeTab,
         inventoryItems: appState.menu.inventory.items,
         worlds: appState.menu.worlds,
-        peers: appState.menu.social.peers
+        peers: appState.menu.social.peers,
+        allItems: state.menu.browse.items
```
Comment (avaer) [inline components/App.js:162]: appState?

File: components/Browse.js [hunk 2]
```diff
@@ -1,8 +1,46 @@
...
+  console.log(props)
+  let allItems = props.allItems || [];
+  return `
+    <div class="twoD-browse">
+      <div class="twoD-browse-list">
+        ${
+          allItems.map((value, index) => {
+            return BrowseCard({
+              id: value.id,
+              name: value.filename || '',
```
Comment (avaer) [inline components/Browse.js:37]: Can this ever be blank? If not it shouldn't have a default.

File: edit.html [hunk 3]
```diff
@@ -10,6 +10,7 @@
   <link href="components/styles/menu.css" rel=stylesheet>
   <link href="components/styles/world.css" rel=stylesheet>
   <link href="components/styles/social.css" rel=stylesheet>
+  <link href="components/styles/browse.css" rel=stylesheet>
```
Comment (avaer) [inline edit.html:13]: Can these CSS files be collapsed? It's already kind of not good that they are separated from the rendering.

Comment (ChrisEddy) [reply edit.html:13]: maybe but not bad to keep separate

File: components/Browse.js (hunk 2 above)
Comment (ChrisEddy) [reply components/Browse.js:37]: could be blank if a bug or something

Comment (avaer) [reply components/Browse.js:37]: But then the bug should be fixed or we should throw. What about value, can that be blank? What about id? Where does it end?

Comment (avaer) [reply components/Browse.js:37]: If this is a legit case we should document why this is possible. If not then there shouldn't be a default.

Comment (ChrisEddy) [reply components/Browse.js:37]: we should check that further up the logic not in the "dumb views", this is just to not error in any situation of using the view

File: edit.html (hunk 3 above)
Comment (avaer) [reply edit.html:13]: It is extra load time.

File: components/Browse.js (hunk 2 above)
Comment (avaer) [reply components/Browse.js:37]: But id can't be blank?

File: edit.html (hunk 3 above)
Comment (ChrisEddy) [reply edit.html:13]: I mean... 200ms~ not that big of a deal imo right now, pre sure the JS takes another 7 secs to complete

xcvbxcvbxcvbxcvb

Comment (avaer) [reply edit.html:13]: It's different on a Quest tethered to a phone.

Comment (ChrisEddy) [reply edit.html:13]: https://www.npmjs.com/package/uglify-js recommend minifying everything into 1 file then

Comment (ChrisEddy) [reply edit.html:13]: 2k line CSS files no fun

Comment (ChrisEddy) [reply edit.html:13]: ill do later as a complete optimization, compression, minify.

Comment (avaer) [reply edit.html:13]: I would just concat them. But yes.

**Pull request merged**